### PR TITLE
Connect to SQL and Create plot Project.

### PR DESCRIPTION
**Pull request by EstyBrandwein**
uploding the changes of connect to SQL,
and creating the plot and project from end to end

### CODE REVIEW

File: BackEnd/BuyRealEstate.API/BuyRealEstate.BL/BL.Implmention/Plot.BL.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using BuyRealEstate.BL.BL.API;
+using BuyRealEstate.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace BuyRealEstate.BL.BL.Implmention
```
Comment (ShoshanaTzi) [inline BackEnd/BuyRealEstate.API/BuyRealEstate.BL/BL.Implmention/Plot.BL.cs:8]: the namespace include 2 BL, why?

File: BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Image.cs [hunk 2]
```diff
@@ -12,5 +12,10 @@ public class Image
         public int ProjectID { get; set; }
         public string ImageDescription { get; set; }
         public byte[] ImageData { get; set; }
+        public DateTime InsertDate { get; set; }
```
Comment (ShoshanaTzi) [inline BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Image.cs:15]: move identical fields to base class (InsertDate, Update, User, State etc.)

File: BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Context.cs [hunk 3]
```diff
@@ -35,9 +36,12 @@ public class BuyRealEstateDbContext : DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            optionsBuilder.UseSqlServer("Server=buyrealestatedb.c1i0c2q8aofb.eu-north-1.rds.amazonaws.com,1433;User Id=BuyRAdmain;Password=liknot_nadlan;");
```
Comment (ShoshanaTzi) [inline BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Context.cs:38]: connection string must be in config file, like `local.settings.json`

File: BackEnd/BuyRealEstate.API/BuyRealEstate.BL/BL.Implmention/Plot.BL.cs (hunk 1 above)
Comment (EstyBrandwein) [reply BackEnd/BuyRealEstate.API/BuyRealEstate.BL/BL.Implmention/Plot.BL.cs]: I changed the project to Core, and there is no this namespace.

File: BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Image.cs (hunk 2 above)
Comment (EstyBrandwein) [reply BackEnd/BuyRealEstate.API/BuyRealEstate.DAL/Models/Image.cs]: we do this.

**Review by TziviDoitch — APPROVED** [review]

**Pull request merged**